task_manager_flutter/lib/ui/auth/login_page.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:task_manager_flutter/components/buttons/tm_elevatebutton.dart';
+import 'package:task_manager_flutter/components/text/tm_text_link.dart';
+import 'package:task_manager_flutter/components/text/tm_title.dart';
+import 'package:task_manager_flutter/components/text_form_field/tm_text_form_filed.dart';
+import 'package:task_manager_flutter/controller/auth/login_controller.dart';
+import 'package:task_manager_flutter/resources/tm_color.dart';
+import 'package:task_manager_flutter/routes/app_page.dart';
+import 'package:task_manager_flutter/utils/validator.dart';
+
+class LoginPage extends GetView<LoginController> {
+  const LoginPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () => FocusScope.of(context).unfocus(),
+      child: Scaffold(
+        body: SafeArea(
+          child: Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: ListView(
+              children: [
+                const TMTitle(title: 'Let’s Login'),
+                const SizedBox(height: 16.0),
+                const TMTitle(
+                  title: 'And notes your idea',
+                  fontSize: 16.0,
+                  fontWeight: FontWeight.w400,
+                  colorTitle: TMColor.onSecondaryBackground,
+                ),
+                const SizedBox(height: 32.0),
```

Comment:
Should create a constant sizedBox32 for SizedBox(height: 32.0) would be a good practice to avoid repetitive code and make your code more maintainable.
ex: const sizedBox32 = SizedBox(height: 32);